applications/PoromechanicsApplication/python_scripts/poromechanics_U_Pw_solver.py

```diff
@@ -385,27 +454,27 @@ def _ConstructSolver(self, builder_and_solver, scheme, convergence_criterion, st
             self.strategy_params.AddValue("desired_iterations",self.settings["desired_iterations"])
             self.strategy_params.AddValue("max_radius_factor",self.settings["max_radius_factor"])
             self.strategy_params.AddValue("min_radius_factor",self.settings["min_radius_factor"])
-            self.strategy_params.AddValue("loads_sub_model_part_list",self.settings["loads_sub_model_part_list"])
+            self.strategy_params.AddValue("loads_sub_model_part_list",self.loads_sub_sub_model_part_list)
             self.strategy_params.AddValue("loads_variable_list",self.settings["loads_variable_list"])
             if nonlocal_damage:
-                self.strategy_params.AddValue("body_domain_sub_model_part_list",self.settings["body_domain_sub_model_part_list"])
+                self.strategy_params.AddValue("body_domain_sub_model_part_list",self.body_domain_sub_sub_model_part_list)
                 self.strategy_params.AddValue("characteristic_length",self.settings["characteristic_length"])
                 self.strategy_params.AddValue("search_neighbours_step",self.settings["search_neighbours_step"])
-                solver = KratosPoro.PoromechanicsRammArcLengthNonlocalStrategy(self.main_model_part,
-                                                                               scheme,
+                solver = KratosPoro.PoromechanicsRammArcLengthNonlocalStrategy(self.computing_model_part,
+                                                                               self.scheme,
                                                                                self.linear_solver,
-                                                                               convergence_criterion,
+                                                                               self.convergence_criterion,
                                                                                builder_and_solver,
                                                                                self.strategy_params,
                                                                                max_iters,
                                                                                compute_reactions,
                                                                                reform_step_dofs,
                                                                                move_mesh_flag)
             else:
-                solver = KratosPoro.PoromechanicsRammArcLengthStrategy(self.main_model_part,
-                                                                       scheme,
+                solver = KratosPoro.PoromechanicsRammArcLengthStrategy(self.computing_model_part,
```

Comment:
I think this name should not be "solver" (the class is already called solver) => it is a solving_strategy

In StructuralMechanics we use "mechanical_solving_strategy" for it